### PR TITLE
Trailing comma for code prettify

### PR DESCRIPTION
**Pull request by sumitra19jha**
**What kind of change does this PR introduce?**
refactoring #753 

**Did you add tests for your changes?**
Not required

**If relevant, did you update the documentation?**
Not required

**Summary**
Indicate trailing comma need

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by palisadoes** [comment]
@sumitra19jha did you work with @nayakastha on this? She's been researching #753. The two of you could have complementary solutions. Collaborate on this, it's not in the spirit of the project if you didn't.

**Comment by sumitra19jha** [comment]
> @sumitra19jha did you work with @nayakastha on this? She's been researching #753. The two of you could have complementary solutions. Collaborate on this, it's not in the spirit of the project if you didn't.

@palisadoes This PR doesn't deal with automatically adding the comma in the end. It is a separate PR which makes sure of the fact that the contributors will add comma in the end to prettify the code for their PR. I haven't written the script with the intention of automatically adding comma which was assigned to @nayakastha. 

But if it voilate the spirit of the project then I am extremely sorry for that.

File: .github/workflows/comma.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    args = arg_parser_resolver()
+    # parses through files and saves to a dict
+    file_names_with_size = {}
+
+    # libPath and testPath dir location
+    lib_path = os.path.expanduser(os.path.join(args.dir, 'lib'))
+    test_path = os.path.expanduser(os.path.join(args.dir, 'test'))
+
+    # counting lines in lib dir
+    for root, _, files in os.walk(lib_path, topdown=False):
```
Comment (palisadoes) [inline .github/workflows/comma.py:34]: You can do the lib dir and test dir together by nesting in another loop

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please work with @nayakastha on creating a comprehensive solution. I've added a suggestion to your coding logic.

File: .github/workflows/comma.py (hunk 1 above)
Comment (sumitra19jha) [reply .github/workflows/comma.py:34]: ok

**Comment by nayakastha** [comment]
Hey! I think this script can be used to solve the issue of trailing commas. But I don't think it will handle corner cases, as there are a lot of them.

**Comment by sumitra19jha** [comment]
What do you mean by corner case?

**Comment by nayakastha** [comment]
> What do you mean by corner case? @nayakastha

Corner cases as in the ones which can't be generalised and have typical usage

**Pull request closed without merging**